### ISSUE TITLE
Feature request: Daum core integration

### ISSUE THREAD

**Issue by EriolGaurhoth** [litchie/dospad#42]
This version of dospad is based around the latest official engine (0.73) which is limited in many ways.  As a recommendation, but completely unsure of how much work it would take to adapt into an ios-capable version, take a look at the source for DOSBOX SVN Daum.  It's the preferred version for anyone who wants to run Windows inside of DOSBOX, includes PCI support, and a host of other features that make DOSBOX more powerful.

http://ykhwong.x-y.net/

**Comment by DominusExult** [comment]
Daum is so filled with patches and wip code (by the dosbox-x fork) that it is no longer stable.

IMO the goal for iOS should be Dos games not the likes of running windows 9x etc. which will take a heavy toll on processing power.

**Comment by EriolGaurhoth** [comment]
That's too bad it's so unstable, I was really hoping to someday run the early Fallout games or Age of Empires on an iPad :(

**Comment by cnnn** [comment]
How about revisiting DOSBox-X, with hugely improved power on 2018 iOS devices than 2 years ago.

**Comment by brunocastello** [comment]
> How about revisiting DOSBox-X, with hugely improved power on 2018 iOS devices than 2 years ago.

I second that. updating the iDOS2 version to the latest SVN + getting the features of DOSBox-X to have windows 98 and 3dfx would be awesome for better gaming on my 2017 12-inch iPad Pro. Processing power isn't an issue because I have been running a different emulation app (UTM, based on QEMU) and some Windows 98 games does work but without 3dfx (no vga device that supports it) and sound (due to a coreaudio issue), though. It's been a breeze to work with Win 9x there; DOSBox-X should be much more faster, even on 2020 (and upcoming) iPad Pros.

I'd do it myself if I knew how to update it with DOSBox-X version.

**Comment by litchie** [comment]
Will look into it.

**Comment by brunocastello** [comment]
> Will look into it.

I’ve contacted the DOSBox-X developer, and here is his reply: https://www.vogons.org/viewtopic.php?p=880609#p880609

**Comment by litchie** [comment]
That's nice.

**Comment by Torinde** [comment]
There is [a fork](https://github.com/imxieyi/dospad) by @imxieyi, but it hasn't been updated in 6 years. 

Also, as mentioned above - it would be better to adopt DOSBox-X (or [Staging](https://github.com/dosbox-staging/dosbox-staging)).

@cnnn mentioned that he made a [DOSPad/DOSBox-X hybrid](https://github.com/joncampbell123/dosbox-x/issues/959#issuecomment-449868345) and DOSBox-X developer said PRs are welcome.

**Comment by DominusExult** [comment]
Dosbox-x has been discontinued AFAIK.

**Comment by Torinde** [comment]
> Dosbox-x has been discontinued AFAIK.

No, [it's regularly updated](https://github.com/joncampbell123/dosbox-x) - more than 1000 commits/year, last week had commits, [last release](https://dosbox-x.com/release-2025.05.03.html) is from May.

**Comment by DominusExult** [comment]
Must have confused it with another fork 🤷‍♂️
Sorry

**Comment by Torinde** [comment]
> Must have confused it with another fork

Maybe [ECE](https://blog.elia.myds.me/?page_id=502).

**Comment by DominusExult** [comment]
Yes, that was it!
